v2/pkg/protocols/common/contextargs/args.go

```diff
@@ -26,6 +28,6 @@ func (Args Args) IsEmpty() bool {
 }
 
 // create a new args map instance
-func newArgs() map[string]interface{} {
-	return make(map[string]interface{})
+func newArgs() *mapsutil.SyncLockMap[string, interface{}] {
+	return &mapsutil.SyncLockMap[string, interface{}]{Map: mapsutil.Map[string, interface{}]{}}
 }
```

Comment:
except `newargs()` all types,methods in this file not used anymore . we can move it contextargs.go and delete this file

Reply:
nice one, forgot to cleanup!